update cennznet api and polkadot api version

KarishmaBothara opened the pull request.

ken-centrality commented on packages/app-accounts/src/Overview.tsx:
```diff
@@ -22,17 +22,17 @@ type SortedAccount = { address: string; isFavorite: boolean };
+//     console.error(error);
+//   }
+// }
 
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/app-accounts/src/Overview.tsx:
```diff
@@ -104,17 +104,17 @@ export default function Overview ({ className, onStatusChange }: Props): React.R
+        {/*    />*/}
+        {/*  </>*/}
+        {/*)}*/}
       </Button.Group>
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/app-explorer/src/BlockInfo/ByHash.tsx:
```diff
@@ -22,11 +23,60 @@ interface Props extends I18nProps {
+  // const events = useCall<KeyedEvent[]>(api.query.system.events.at, [value]);
+  // const getBlock = useCall<SignedBlock>(api.rpc.chain.getBlock, [value]);
+  // const getHeader = useCall<HeaderExtended>(api.derive.chain.getHeader, [value]);
+
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/app-explorer/src/BlockInfo/ByHash.tsx:
```diff
@@ -22,11 +23,60 @@ interface Props extends I18nProps {
+  // const blockNumber = getHeader?.number.unwrap();
+  // const parentHash = getHeader?.parentHash.toHex();
+  // const hasParent = !getHeader?.parentHash.isEmpty;
 
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/app-settings/src/General.tsx:
```diff
@@ -101,17 +101,17 @@ export default function General ({ className, isModalContent, onClose }: Props):
+          {/*    />*/}
+          {/*  </div>*/}
+          {/*)}*/}
           <div className='ui--row'>
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/apps/webpack.config.js:
```diff
@@ -120,74 +132,90 @@ function createWebpack ({ alias = {}, context, name = 'index' }) {
+      //   ),
+      //   PAGE_TITLE: 'CENNZnet Portal'
+      // }),
+      new webpack.optimize.SplitChunksPlugin(),
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/react-api/src/index.ts:
```diff
@@ -18,8 +18,8 @@ export {
+  // getLedger,
+  // isLedger,
+  // isLedgerCapable
 };
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/react-api/src/ledger.ts:
```diff
@@ -1,59 +1,60 @@
+//   }
+//
+//   return ledger as Ledger;
+// }
```
Looks like the majority of `react-api/src/staking` files are commented out, worth to remove them to keep things tidy?

ken-centrality commented on packages/react-components/src/Call.tsx:
```diff
@@ -6,24 +6,38 @@ import BN from 'bn.js';
+//   withHash?: boolean;
+//   tip?: BN;
+// }
+
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/react-components/src/Call.tsx:
```diff
@@ -37,24 +51,70 @@ interface Value {
+    //   ? value.hash
+    //   : null;
+    //
+    // setExtracted({ hash, params, values });
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/app-address-book/src/Address.tsx:
```diff
@@ -142,10 +142,15 @@ function Address ({ address, className, isFavorite, toggleFavorite }: Props): Re
+        {/*  className={`${isFavorite && 'isSelected'}`}*/}
+        {/*  name={isFavorite ? 'star' : 'star outline'}*/}
+        {/*  onClick={_onFavorite}*/}
+        {/*/>*/}
```
Remove the comment block if we don't need it anymore

ken-centrality commented on packages/app-address-book/src/Address.tsx:
```diff
@@ -146,11 +146,11 @@ function Address ({ address, className, isFavorite, toggleFavorite }: Props): Re
+        {/*  name={isFavorite ? 'star' : 'star outline'} */}
+        {/*  onClick={_onFavorite} */}
+        {/* /> */}
       </td>
```
Remove the comment block if we don't need it anymore

ken-centrality reviewed (changes requested):
Looks good, there quite a few commented out code blocks, would be great if we can tidy them up before merge.

I noticed we've bumped up to use node `16`, then it's worth to update `.nvmrc` to `lts/gallium` to match with the requirement.

Also, does this resolve #170 and #172?

KarishmaBothara commented:
> Looks good, there quite a few commented out code blocks, would be great if we can tidy them up before merge.
> 
> I noticed we've bumped up to use node `16`, then it's worth to update `.nvmrc` to `lts/gallium` to match with the requirement.
> 
> Also, does this resolve #170 and #172?

thanks for looking thro' the PR.. I removed commented code blocks and updated .nvmrc
Yes, updated the code to resolve #170 and #172

ken-centrality commented:
Looks good, but can you update .nvmrc to be `lts/gallium` instead of `16.0`, that will lock it to the LTS version of node 16, which is `16.13` atm

KarishmaBothara commented:
> lts/gallium

thanks.. done

KarishmaBothara commented:
> Looks good, but can you update .nvmrc to be `lts/gallium` instead of `16.0`, that will lock it to the LTS version of node 16, which is `16.13` atm

thanks done

holygits commented:
@KarishmaBothara shall we merge?

holygits commented:
need this so we can send txs to rata post update

KarishmaBothara commented:
> @KarishmaBothara shall we merge?

yes.. wll check the last four steps of the pipeline.. not sure why status is waiting for that
